.gitmodules

```diff
@@ -77,3 +77,7 @@
 	path = src/jaegertracing/thrift
 	url = https://github.com/apache/thrift.git
 	branch = 0.13.0
+[submodule "src/libkmip"]
+	path = src/libkmip
+	url = https://github.com/ceph/libkmip
+	branch = wip-morestuff
```

Comment:
i'd suggest use a branch with a more meaningful name so

- it is less likely to be removed be accident
- we can switch to another branch when we need to bump up the submodule in case we the new branch is not a fast-forward of the old branch.